addons/test_mail_full/tests/test_rating.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.test_mail_full.tests.common import TestMailFullCommon, TestMailFullRecipients
+from odoo.tests.common import users
+from odoo.tools import mute_logger
+from odoo.tests import tagged
+
+
+@tagged('rating')
+class TestRatingCommon(TestMailFullCommon, TestMailFullRecipients):
```

Comment:
can you merge it with `TestRatingFlow` ? (look like it's only used in this file)